RELEASING.md

```diff
@@ -0,0 +1,29 @@
+# Release Checklist
+
+- [ ] check tests pass on [GitHub Actions](https://github.com/python/cherry-picker/actions)
+      [![GitHub Actions status](https://github.com/python/cherry-picker/actions/workflows/main.yml/badge.svg)](https://github.com/python/cherry-picker/actions/workflows/main.yml)
+
+- [ ] Update [changelog](https://github.com/python/cherry-picker#changelog)
+
+- [ ] Go to https://github.com/python/cherry-picker/releases
+
+- [ ] Click "Draft a new release"
+
+- [ ] Click "Choose a tag"
+
+- [ ] Type the next `cherry-picker-vX.Y.Z` version and select "**Create new tag: cherry-picker-vX.Y.Z** on publish"
+
+- [ ] Leave the "Release title" blank (it will be autofilled)
+
+- [ ] Click "Generate release notes" and amend as required
+
+- [ ] Click "Publish release"
+
+- [ ] Check the tagged `GitHub Actions build <https://github.com/python/cherry-picker/actions/workflows/deploy.yml>`
+      has deployed to `PyPI <https://pypi.org/project/cherry_picker/#history>`
+
+- [ ] Check installation:
+
+```bash
+pip3 uninstall -y cherry_picker && pip3 install -U cherry_picker && cherry_picker --version
```

Comment:
On the readme.rst, we use
`python -m pip install ...` instead of `pip3 install`.
I think the instructions should match.

Reply:
Good idea!

https://snarky.ca/why-you-should-use-python-m-pip/